<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compiler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>341</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Source File</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>611</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit_1">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(225,225,225,180);
border-style:outset;
border-width:2px;
border-color:rgba(0,0,0,30);
color:rgba(0,0,0,200);
padding:6px;}
QPushButton:pressed{
background-color:rgba(205,230,255,200);
border-color:rgba(0,120,215,30);
border-style:inset;
color:rgba(0,0,0,100);}
QPushButton:hover{
background-color:rgba(230,240,250,100);
border-color:rgba(0,120,215,200);
color:rgba(0,0,0,200);}
</string>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(225,225,225,180);
border-style:outset;
border-width:2px;
border-color:rgba(0,0,0,30);
color:rgba(0,0,0,200);
padding:6px;}
QPushButton:pressed{
background-color:rgba(205,230,255,200);
border-color:rgba(0,120,215,30);
border-style:inset;
color:rgba(0,0,0,100);}
QPushButton:hover{
background-color:rgba(230,240,250,100);
border-color:rgba(0,120,215,200);
color:rgba(0,0,0,200);}
</string>
    </property>
    <property name="text">
     <string>Execute SDT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(225,225,225,180);
border-style:outset;
border-width:2px;
border-color:rgba(0,0,0,30);
color:rgba(0,0,0,200);
padding:6px;}
QPushButton:pressed{
background-color:rgba(205,230,255,200);
border-color:rgba(0,120,215,30);
border-style:inset;
color:rgba(0,0,0,100);}
QPushButton:hover{
background-color:rgba(230,240,250,100);
border-color:rgba(0,120,215,200);
color:rgba(0,0,0,200);}
</string>
    </property>
    <property name="text">
     <string>Generate MIPS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
